build.sbt

```diff
@@ -22,17 +22,22 @@ addCommandAlias("measure-all", "benchmarks/jmh:run -rff results.csv")
 addCommandAlias("measure", "benchmarks/jmh:run -rff results.csv .*parsebackRun")
 addCommandAlias("profile", "benchmarks/jmh:run -prof jmh.extras.JFR -f 1 .*parsebackRun")
 
+publishGithubUser in ThisBuild := "djspiewak"
+publishFullName in ThisBuild := "Daniel Spiewak"
+
 lazy val root = project
   .in(file("."))
   .settings(name := "root")
   .settings(noPublishSettings)
-  .aggregate(benchmarks, coreJVM, coreJS)
```

Comment:
poor benchmarks is being held back by `gll-combinators`. If I get time I'll head there too to help with the push, but for now I just took benchmarks out of the root picture and set it's scalaVersion to be the good ole' fashioned 2.11.

I can see that as being a controversial move, so looking forward to any thoughts on this.